grpc/server/protobuf/casper/ipc.proto

```diff
@@ -453,8 +453,8 @@ message ChainSpec {
     }
 
     message ActivationPoint {
-        // equal to Block.Header.rank
-        uint64 rank = 1;
+        // equal to Block.Header.height
+        uint64 height = 1;
```

Comment:
Just FYI, when you're [updating](https://developers.google.com/protocol-buffers/docs/proto3#updating) protobuf definitions it's a good practice to [`reserve`](https://developers.google.com/protocol-buffers/docs/proto3#reserved) field number and add a new one. 

This is not the case here though, since we won't get messages with the old definition.